astropy/logger.py

```diff
@@ -19,6 +19,14 @@
 
 __all__ = ['Conf', 'conf', 'log', 'AstropyLogger', 'LoggingError']
 
+# import the logging levels from logging so that one can do:
+# log.setLevel(log.DEBUG), for example
+logging_levels = ['DEBUG', 'CRITICAL', 'FATAL', 'ERROR', 'WARN', 'WARNING',
```

Comment:
Let's not import WARN, and just use WARNING.  I think the fact that both exist is of historical interest, but as far as importing into Astropy goes I don't think we need both.  I think also you should list these in their numerical order, so DEBUG comes after INFO, etc.  Just a minor nit.